.github/dependabot.yml

```diff
@@ -16,7 +16,7 @@ updates:
   - package-ecosystem: "npm"
     directory: "/frontend"
     schedule:
-      interval: "weekly"
```

Comment:
New green check mark for valid find/hide opts:

Before:
![Screenshot from 2022-05-03 12-56-42](https://user-images.githubusercontent.com/6226732/166502280-95d47cd3-174b-44bd-b838-90799f2c6021.png)

After:
![Screenshot from 2022-05-03 12-56-51](https://user-images.githubusercontent.com/6226732/166502309-76174f4b-3a62-417e-b8b8-1f50edc0c8af.png)